src/test/java/use_cases/main_menu/SessionSaveUCITest.java

```diff
@@ -0,0 +1,63 @@
+package use_cases.main_menu;
+
+import entities.SessionStorage;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import views.file_session_storage.FileSessionStorage;
+import views.main_menu.MainMenuP;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Tests the SessionSaveUCI class
+ */
+class SessionSaveUCITest {
+    static SessionStorage sampleSession;
+
+    @BeforeAll
+    public static void SessionLoadUCICreateSampleSession() {
+        sampleSession = new SessionStorage();
+        sampleSession.addRecurExpense(new entities.Expense("abc", new entities.Category("lala", 10.0), 3.0));
+        sampleSession.addMonth(new entities.MonthlyStorage(1, 20.0));
+    }
+
+    /**
+     * Tests saving a session into a file correctly
```

Comment:
Instead of saying "correctly" and "failure", you should specify what EXACTLY makes it a success or fail. For example, a clearer Javadoc could be "Tests saving a session with a valid filename". Also, you should specify the @throws parameters and why you are throwing them (i.e. you are providing a current filename, so this error should not throw).

Reply:
Rewrote in new commit